components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/openidconnect/DefaultIDTokenBuilder.java

```diff
@@ -657,13 +658,14 @@ protected String signJWTWithRSA(JWTClaimsSet jwtClaimsSet,
     }
 
     /**
-     * @param request
+     * @param accessToken
      * @return AuthorizationGrantCacheEntry contains user attributes and nonce value
      */
-    private AuthorizationGrantCacheEntry getAuthorizationGrantCacheEntry(OAuthTokenReqMessageContext request) {
-        String authorizationCode = getAuthorizationCode(request);
-        AuthorizationGrantCacheKey authorizationGrantCacheKey = new AuthorizationGrantCacheKey(authorizationCode);
-        return AuthorizationGrantCache.getInstance().getValueFromCacheByCode(authorizationGrantCacheKey);
+    private AuthorizationGrantCacheEntry getAuthorizationGrantCacheEntryFromToken(String accessToken) {
```

Comment:
Add a new line after method signature. This should be applied with formatting

Reply:
Added.

Reply:
Added